myuw/dao/emaillink.py

```diff
@@ -0,0 +1,28 @@
+"""
+This class encapsulates the mapping between
+email address and service provider login
+"""
+
+import re
+import os
+import csv
+
+
+class EmailServiceUrlException(Exception):
```

Comment:
I like having a custom exception, but there's a general convention (used at least in restclients) to put them in exceptions.py.  I'd like to use that convention in myuw.